src/Management/src/Endpoint/Hypermedia/Links.cs

```diff
@@ -22,7 +22,9 @@ public class Links
 #pragma warning disable SA1300 // Element should begin with upper-case letter
 #pragma warning disable S100 // Methods and properties should be named in PascalCase
 #pragma warning disable IDE1006 // Naming Styles
+#pragma warning disable S4004 // Collection properties should be readonly
```

Comment:
Oh, _wow_, do we really need 4 rule suppressions here? That's getting pretty wild

Reply:
I wonder why this public property is named `_links`. I probably added the suppressions to avoid a breaking change. Can't we just rename it to `Links`?

Reply:
This is for serialization, will replace with a JsonPropertyName instead